parlai/mturk/core/mturk_data_handler.py

```diff
@@ -729,6 +729,7 @@ def get_all_task_pairings_for_worker(self, worker_id, task_group_id=None):
 
     @staticmethod
     def get_conversation_data(task_group_id, conv_id, worker_id, is_sandbox):
+        '''A poorly named function that gets conversation data for a worker'''
```

Comment:
is, e.g., `get_worker_conversation_data` too long of a name? or would changing this require some amount of nontrivial refactoring elsewhere

Reply:
Nontrivial, will do in a follow-up

Reply:
nit: we use double quotation marks for docstrings elsewhere

Reply:
also lol at the description 😆

Reply:
Agh I have to stop making that mistake!